internal/kafka/internal/services/kafka.go

```diff
@@ -1261,3 +1273,25 @@ func (k *kafkaService) AssignBootstrapServerHost(kafkaRequest *dbapi.KafkaReques
 
 	return nil
 }
+
+// getRoute53RegionFromKafkaRequest calculates the AWS region to be used for
+// Route53 from the kafka request. It calculates its value using the
+// cloud provider specified in the kafka request.
+// Route53 is a global service which means that in most of the cases
+// the region specified is only used to access a regional endpoint in AWS.
+// There are some parts of the Route53 functionality that are regional.
+// For what we perform which is create hosted zones and entries in them
+// that is a global functionality.
+// See: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/disaster-recovery-resiliency.html
+// If at some point we end up needing Route53 regional functionalities this
+// mechanism should be reevaluated
+func (k *kafkaService) getRoute53RegionFromKafkaRequest(kafkaRequest *dbapi.KafkaRequest) (string, error) {
+	switch kafkaRequest.CloudProvider {
+	case cloudproviders.AWS.String():
+		return aws.DefaultAWSRoute53Region, nil
```

Comment:
Quick question: Will this affect the deletion of existing Kafkas (whose region is different from `us-east-1`) i.e there won't be any issues with deleting an already created CNAMEs on other (different from the default) AWS regions? 

Based on the comment on this method, I think we should be good but thought I would ask to get confirmation.

@miguelsorianod

Reply:
We should be good